my-app/src/Components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,33 @@
+import {ReactComponent as Light} from '../../assets/light.svg';
+import {ReactComponent as Dark} from '../../assets/dark.svg';
+import { Link, NavLink } from "react-router-dom";
+import style from './Navbar.module.scss';
+import User from "../User/User";
+import { useDispatch, useSelector } from 'react-redux';
+import { switchTheme } from '../../store/themeSlice';
+import {toggleActive} from '../../store/activeSlice';
+import InputOutput from '../InputOutput/InputOutput';
+
+const Navbar = () => {
+    const {isActive} = useSelector((state) => state.active);
+    const dispatch = useDispatch();
+    return (
+        <div className={!isActive ? style.navbar : `${style.navbar} ${style.active}`}>
+            <div className = {style.menuBox}>
+                <NavLink to = '/user' className = {style.userPoint} onClick={() => dispatch(toggleActive())}><User userName= {'Artem Malkin'}/></NavLink>
```

Comment:
create one function const  toggleSideBar = () => dispatch(toggleActive()) and use it for all navlinks